test/user-test.js

```diff
@@ -64,7 +64,8 @@ describe('User', function() {
 
   it('should be able to decide to cook a recipe', function() {
     user.decideToCook(recipe);
-    expect(user.recipesToCook[0].name).to.equal('Rice bowl with Fried Egg');
+    user.removeFromRecipesToCook(recipe);
```

Comment:
this could be a great opportunity for sad path testing to make sure it is accessing the correct recipe name .